rockcraft.yaml

```diff
@@ -176,29 +176,29 @@ parts:
       - procps
       - openjdk-11-jre-headless
       - python3-setuptools
-    overlay-script: |
-      SPARK_GID=185
-      SPARK_UID=185
 
-      # Create a user in the $CRAFT_OVERLAY chroot
-      groupadd -R $CRAFT_OVERLAY -g ${SPARK_GID} spark
-      useradd -R $CRAFT_OVERLAY -m -r -g spark -u ${SPARK_UID} spark
     override-prime: |
-      SPARK_GID=185
-      SPARK_UID=185
+      SPARK_GID=584792
```

Comment:
It might be worth adding a code comment linking to an explanation of this choice of GID and UID (https://discourse.ubuntu.com/t/unifying-user-identity-across-snaps-and-rocks/36469)